var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -29,15 +29,18 @@ class Ginkgo(CMakePackage, CudaPackage):
     depends_on('cmake@3.9:', type='build')
     depends_on('cuda@9:',    when='+cuda')
```

Comment:
Will the CudaArchitectureSelector work properly here ? Also, I don't think there are any tests which have `cuda` as a spec. Maybe I missed something ?